skt/__init__.py

```diff
@@ -468,12 +467,8 @@ def bisect_iter(self, bad):
 class kbuilder(object):
     def __init__(self, path, basecfg, cfgtype=None, makeopts=None,
                  enable_debuginfo=False):
-        # FIXME Move expansion up the call stack, as this limits the class
-        # usefulness, because tilde is a valid path character.
-        self.path = os.path.expanduser(path)
-        # FIXME Move expansion up the call stack, as this limits the class
-        # usefulness, because tilde is a valid path character.
-        self.basecfg = os.path.expanduser(basecfg)
+        self.path = path
+        self.basecfg = basecfg
```

Comment:
Those are fixed up in 83862e8 and d02ba69.

Reply:
This would make the all the previous patches break skt, which is what we're trying to avoid with the rule "every commit works". This also makes review difficult.